app/src/main/java/com/eveningoutpost/dexdrip/BGHistory.java

```diff
@@ -2,22 +2,16 @@
 
```

Comment:
I think the changes shown here are recent ones where your branch was out of sync. Can you checkout the current version of this and add that to nullify the changes to BGHistory.java